--- openal-soft-1.21.0-mac-64/config.h.in	2020-11-04 12:00:35
+++ openal-soft-1.21.0-mac-64/config.h.in-new	2023-09-30 22:42:10
@@ -1,6 +1,6 @@
 /* API declaration export attribute */
-#define AL_API  ${EXPORT_DECL}
-#define ALC_API ${EXPORT_DECL}
+#define AL_API  __attribute__((visibility("hidden")))
+#define ALC_API __attribute__((visibility("hidden")))
 
 /* Define if HRTF data is embedded in the library */
 #cmakedefine ALSOFT_EMBED_HRTF_DATA
